torch/csrc/jit/runtime/jit_trace.cpp

```diff
@@ -290,6 +290,14 @@ std::shared_ptr<Graph> TraceGraph(std::shared_ptr<Graph> graph, Stack& stack) {
     ni->setType(ni->type());
     td.old_to_new_[inp] = ni;
   }
+  // type inputs
```

Comment:
nit: you already checked in a fix ?

Reply:
That diff (D32382958) just landed. Will update this PR.